stylesheets/main.css

```diff
@@ -7,7 +7,7 @@
 
 	/* shuttle config */
 	--shuttle-logo: var(--shuttle-logo-jpl);
-	--rocket-fuel: 0;
+	--rocket-fuel: 11;
```

Comment:
I think this needs to be set to `1` in order to work. So you could do, --rocket-fuel: 1;